src/styles/styles.js

```diff
@@ -290,18 +290,16 @@ const styles = {
 
     picker: {
         inputIOS: {
-            fontFamily: fontFamily.GTA,
-            fontSize: variables.fontSizeNormal,
-            paddingLeft: 12,
-            paddingRight: 12,
-            paddingTop: 10,
-            paddingBottom: 10,
-            borderRadius: variables.componentBorderRadius,
+            flex: 1,
             borderWidth: 1,
+            borderRadius: variables.componentBorderRadiusNormal,
             borderColor: themeColors.border,
-            color: themeColors.text,
-            height: variables.inputComponentSizeNormal,
-            opacity: 1,
+            paddingTop: 25,
+            paddingHorizontal: 12,
+            paddingVertical: 8,
+            justifyContent: 'center',
+            height: '100%',
+            backgroundColor: '#fff',
```

Comment:
You should only ever reference `themeColors`, not use hex colors directly.

Reply:
Done